students/M_Sunday/lesson05/Mailroom_Part_3.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/python
+from operator import itemgetter
+
+# Lesson 05, Mailroom Part 3 Assignment
+
+
+# ------- Global Assignments -------
+donor_data = {'John Marks': [5000.34, 2500, 1267],
+              'Matt Smith': [750],
+              'Kelsey Rodgers': [55.76],
+              'John Williams': [500, 1898],
+              'Morgan Stanley': [12000.98, 8670],
+              'Roger Johnson': [264, 942.23],
+              'Sherry Wilson': [4000, 2324.1, 1352],
+              'Alex Grant': [8739, 4312]}
+# ----------------------------------
+
+
+def menu_display():
+    print("\n\n----- MENU -----\n"
+          "1: Send a Thank You\n"
+          "2: Create a Report\n"
+          "3: Send letters to everyone\n"
+          "4: Quit\n")
+    menu_sel_ = input("------> "
+                      "Select an option from the menu: ")
+    return menu_sel_
+
+
+def list_display():
+    print("\n---- List of all Donors ----")
+    [print("- {}".format(item)) for item in donor_data]
```

Comment:
it is not recommend to use comprehensions purely as a shortcut and to ignore returned data